scripts/known_constants.txt

```diff
@@ -0,0 +1,2280 @@
+CONST_THR_SPINFUSOR                                      1
+CONST_THR_SMG                                            2
```

Comment:
Where/how did you get this list? I have to ask because I can only accept code for which you are the copyright holder (or someone else who consents to integration in this AGPL3 project).

Reply:
This is actually an interesting problem which I've asked a number of people about and haven't found a clear answer.

Would prefer to discuss on a private channel if possible - could we make a new private channel on the discord now that #development is public?